ui/css/treeherder-job-buttons.css

```diff
@@ -105,8 +105,13 @@
   border: 2px solid;
   border-radius: 3px;
   font-size: 12px;
-  transform: scale(1.7, 1.7);
   z-index: 1;
+  transition: 0.1s;
+  transform: scale(1.1, 1.1);
```

Comment:
A scale of ``1.1`` is going to be to subtle to pick out from the rest of the job field.  Let's go with ``1.3``.

Reply:
Fixed